src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -285,10 +223,7 @@ public function selectShippingAddressFromAddressBook(AddressInterface $address)
         JQueryHelper::waitForFormToStopLoading($this->getDocument());
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function selectBillingAddressFromAddressBook(AddressInterface $address)
+    public function selectBillingAddressFromAddressBook(AddressInterface $address): void
```

Comment:
and here 233 and 237 replace `.item[data-id="%s"]` for example: `[data-test-item-id="%s"]`